page-components/buttons.md

```diff
@@ -165,8 +159,9 @@ is functioning as intended.
 
 {:/nomarkdown}
 </div>
+<br>
 
-## States
+## Primary button states
```

Comment:
The "States" link on the intra-page navigation is now broken, because of the renaming and breaking apart of the different button types. Maybe we keep "States" as the main category, and the button types within the category?